src/components/dar_dataset_table/DarDatasetTableCellData.js

```diff
@@ -0,0 +1,71 @@
+import {styles} from './DarDatasetTable';
+
+export function dataUseGroupCellData({dataUseGroup, label= 'data-use'}) {
+  return {
+    data: dataUseGroup,
+    id: dataUseGroup,
+    style : {
+      color: '#354052',
+      fontSize: styles.fontSize.dataUseGroup,
+      fontWeight: 'bold',
+      paddingRight: '2%'
+    },
+    label
+  };
+}
+
+export function votesCellData({elections, votes, dataUseGroup, label= 'votes'}) {
+  const dataAccess = elections?.find((election) => election.electionType === 'DataAccess');
+
+  let status = dataAccess?.status || 'N/A';
+
+  if (status === 'Closed') {
+    status = votes?.finalVotes?.some((v) => v.vote === true) ? 'Approved' : 'Denied';
+  }
```

Comment:
I believe it is, but I just want to check that this is sound logic for the status of a bucket.

Reply:
If any final vote is true, then it is definitely `Approved`. If any final vote is false, then it is definitely `Denied`

Reply:
So, would it just be 'In Progress' then?

Reply:
Another possibility is `Canceled` where an election was closed with no final vote values. I think the logic should be something like this:

If any final votes are non-null && true -> `Approved`
If any final votes are non-null && false -> `Denied`
Else - there is no final vote status to report.

Reply:
The above logic is a little too loose in that there's only Approved and Denied, but we do have that third case.

Reply:
Right now, I am displaying `Canceled`, `Open`, etc. as well since it's deriving the status from the original election status. In this table, do we prefer just showing `Approved` and `Denied`, or do we want to be more specific in the case neither is true, e.g. `Open` `Canceled` etc.